### PR TITLE
Added doc for creating basic kueue resources

### PR DESCRIPTION
**Pull request by Ygnas**
# Issue link

https://issues.redhat.com/browse/RHOAIENG-6659

# What changes have been made

# Verification steps

## Checks
- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [x] Testing is not required for this change

### CODE REVIEW

**Review by ChristianZaccaria — CHANGES_REQUESTED** [review]
Great work Ignas, thanks! A few small nitpicks

File: docs/setup-kueue.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+When setting up a Cluster Queue in Kueue, it's crucial that the resource specifications match the actual capacities and operational requirements of your cluster. The example provided outlines a basic setup; however, each cluster may have different resource availabilities and needs.
+```yaml
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ClusterQueue
+metadata:
+  name: "cluster-queue"
+spec:
+  namespaceSelector: {} # match all.
+  resourceGroups:
+  - coveredResources: ["cpu", "memory", "pods"]
```
Comment (ChristianZaccaria) [inline docs/setup-kueue.md:31]:
```suggestion
  - coveredResources: ["cpu", "memory", "pods", "nvidia.com/gpu"]
```

**Review by ChristianZaccaria — CHANGES_REQUESTED** [review]

File: docs/setup-kueue.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+## 3. Local Queue (With Default Annotation):
+A Local Queue represents a queue associated with a specific namespace within the cluster. It allows namespace-level control over workload prioritization and resource allocation.
+```yaml
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: LocalQueue
+metadata:
+  namespace: team-a
+  name: team-a-queue
+  annotations:
+    kueue.x-k8s.io/default-queue: "true"
```
Comment (Bobbins228) [inline docs/setup-kueue.md:56]: Perhaps a line below explaining why a cluster admin would want to set the default annotation.

**Review by Bobbins228 — CHANGES_REQUESTED** [review]

File: docs/setup-kueue.md (hunk 2 above)
Comment (Ygnas) [reply docs/setup-kueue.md:56]: Added the line below it.

**Pull request merged**